samples/azure/blob-storage-databus/ABSDataBus_1/Sender/Program.cs

```diff
@@ -22,7 +22,8 @@ static void Main()
 
         #endregion
 
-        endpointConfiguration.UsePersistence<InMemoryPersistence>();
+        endpointConfiguration.UseTransport<LearningTransport>();
+        endpointConfiguration.UsePersistence<LearningPersistence>();
```

Comment:
Is UsePersistence mandatory here? this receiver doesn't seem to use persistance, there are no saga's and the  learning transport does not require a persister. Imo this line can be removed.